src/numberVerification.js

```diff
@@ -0,0 +1,15 @@
+export default function numberVerification(number) {
+  const array = number.split(',');
+
+  if (array.length !== 6)
+    throw new Error('[ERROR] : 숫자가 잘못된 형식입니다.');
+
+  const arrayNumber = array.map((item) => Number(item));
+
+  for (let i = 0; i < arrayNumber.length; i++) {
```

Comment:
에어비앤비에서는 증감연산자를 지양하라고 합니다!
https://github.com/airbnb/javascript#variables--unary-increment-decrement

```suggestion
  for (let i = 0; i < arrayNumber.length; i += 1) {
```